server/src/main/java/app/auth/AuthenticationController.java

```diff
@@ -15,21 +15,21 @@ public class AuthenticationController {
 
     @PostMapping("/register")
     public AuthenticationResponse register(
-        @RequestBody RegistrationRequest request
+            @RequestBody RegistrationRequest request
     ) {
         return service.register(request);
     }
 
     @PostMapping("/authenticate")
     public AuthenticationResponse authenticate(
-        @RequestBody AuthenticationRequest request
+            @RequestBody AuthenticationRequest request
     ) {
         return service.authenticate(request);
     }
 
     @PostMapping("/refresh-token")
     public AuthenticationResponse refresh(
```

Comment:
please use this codestyle 
<img width="1299" alt="Screenshot 2024-12-14 at 23 57 47" src="https://github.com/user-attachments/assets/5af02e3c-fc2a-4f23-8a69-40a751ec0575" />

in server/config/intelij-code-style

that i have added in last pr.